src/main/java/eu/tornplayground/tornapi/models/faction/attacks/Modifiers.java

```diff
@@ -0,0 +1,67 @@
+package eu.tornplayground.tornapi.models.faction.attacks;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import eu.tornplayground.tornapi.models.Model;
+
+import java.util.Objects;
+
+public class Modifiers extends Model {
```

Comment:
`warlord_bonus` is missing

[Unofficial docs](https://tornapi.tornplayground.eu/faction/attacks#attack_result)

Reply:
added it to the class and the tests